department-of-veterans-affairs/caseflow
# VACOLS upload success page

Connects #5823 

### Description
This PR adds a success page for the VACOLS upload. The page doesn't quite match the mock because of this issue that will be fixed in a different PR: https://github.com/department-of-veterans-affairs/caseflow/issues/6229

### Testing Plan
1. Go to hearings/schedule/build
1. Go through flow 1 twice, once with an ro/co upload and then with a judge upload
1. Confirm that the pages all render correctly (particularly the success page)
